backend/apid/middlewares/refresh_token.go

```diff
@@ -44,6 +44,19 @@ func (m RefreshToken) Then(next http.Handler) http.Handler {
 			return
 		}
 
+		// Retrieve the claims for the access token
+		accessClaims, err := jwt.GetClaims(accessToken)
+		if err != nil {
+			logger.WithError(err).Error("could not parse the access token claims")
+			http.Error(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+
+		if accessClaims.Provider.ProviderType == "oidc" {
```

Comment:
Should "oidc" be a constant, and should we accept caps spelling?

Reply:
The actual constant is defined in the enterprise OIDC provider, here:
https://github.com/sensu/sensu-enterprise-go/pull/628/files#diff-ef44a838326105c4d512ea9ba2598a2bR12
But we obviously cannot import enterprise into core!

I suppose we can add a constant at the top of this `refresh_token.go` file? Not sure it would make a big difference since it still wouldn't track the constant that's defined in enterprise.

Cap spelling is not an issue; it should always be lowercase as defined by the constant.